src/main/java/scrublords/entities/enemies/LevelOneBoss.java

```diff
@@ -0,0 +1,37 @@
+package scrublords.entities.enemies;
+
+import scrublords.core.CollisionDetection;
+import scrublords.core.SpriteDimensions;
+import scrublords.core.SpriteSheet;
+import scrublords.core.Visualization;
+import scrublords.entities.core.*;
+import scrublords.tilemaps.TileMap;
+
+import java.util.ArrayList;
+
+/**
+ * @author Nikolay Zahariev <nikolay.g.zahariev@gmail.com>.
+ */
+public class LevelOneBoss {
```

Comment:
Give it a name because we might want to use it somewhere else and the naming will for sure sound wrong. For example, it sounds wrong to call LevelOneBoss in the Endless Level.